pom.xml

```diff
@@ -63,7 +63,6 @@
     <module>resteasy</module>
     <module>resteasy-genericresponse</module>
```

Comment:
We should also be able to remove `<swagger-core.version>1.3.12</swagger-core.version>`

Reply:
removed